### DISCUSSION TITLE
Hi I'm back again :D

### DISCUSSION THREAD

**Discussion by ghost** [hydro-dev/Hydro#230] [Q&A] [answered]
Our School (somewhere in Wisconsin) blocked spoj.com ,but there's quite a lot neat problems.
Your VJudge can configure CodeForces, so can you add SPOJ support?
thx anyway.

**Comment by undefined-moe** [comment, marked as answer]
SPOJ already supported. You can see it in sourcecode. Configuration is similar to Codeforces.
